File: server/src/main/java/com/fluffy/exam/ui/request/UpdateExamDescriptionWebRequest.java
package com.fluffy.exam.ui.request;

import com.fluffy.exam.application.request.UpdateExamDescriptionRequest;
import com.fluffy.global.web.Accessor;

public record UpdateExamDescriptionWebRequest(String description) {

    public UpdateExamDescriptionRequest toAppRequest(Long examId, Accessor accessor) {
        return new UpdateExamDescriptionRequest(description, examId, accessor);
    }
}

File: server/src/main/kotlin/com/fluffy/exam/ui/request/UpdateExamDescriptionWebRequest.kt
package com.fluffy.exam.ui.request

import com.fluffy.exam.application.request.UpdateExamDescriptionRequest
import com.fluffy.global.web.Accessor

data class UpdateExamDescriptionWebRequest(val description: String) {

    fun toAppRequest(examId: Long, accessor: Accessor): UpdateExamDescriptionRequest {
        return UpdateExamDescriptionRequest(
            examId = examId,
            description = description,
            accessor = accessor,
        )
    }
}
